### DISCUSSION TITLE
How To Turn Off Permalink Auto Conplete when edit on modal

### DISCUSSION THREAD

**Discussion by moathOSA** [area17/twill#1571] [Q&A] [answered]
when trying to edit the title from the header section 
![Screenshot 2022-04-07 at 11-06-15 Helium Admin](https://user-images.githubusercontent.com/60176683/162151330-329e01ba-c15c-480b-a236-cb3e3ab2399e.png)

the permalink will take the same change the the title take  , i want to turn of aut0 complete when edit the title from the modal

**Comment by shawkiIIalasy** [comment, marked as answer]
Hello Moath, 

you can control create fields by override  `create.balde.php`.

Override create partials https://twill.io/docs/guides/adding-fields-to-the-create-modal.html#customize-with-more-control

Also check @pboivin comment on this question https://github.com/area17/twill/discussions/983#discussioncomment-888652
